EPRN.Portal/Views/Prns/Create.cshtml

```diff
@@ -85,7 +85,7 @@
                                         }
                                         else if (row.Tonnage.DisplayWarningForTonnage())
                                         {
-                                            @CreatePrnPern.BalanceWarningPrn
+                                            @CreatePrnPern.BalanceWarningPern
```

Comment:
Should this be BalanceWarningPrn? I.e. the balance warning for a reprocessor rather than exporter?

Reply:
Done.